### PR TITLE
Fix deprecation warnings, update tooling

### PR DESCRIPTION
**Pull request by ngnpope**
Fixed a few deprecation warnings that we came across and updated everything similar to what was done for `tg-react`.

Feel free to make any adjustments.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,8 +5,8 @@ tg-utils
 .. image:: https://img.shields.io/pypi/v/tg-utils.svg
         :target: https://pypi.python.org/pypi/tg-utils
 
-.. image:: https://travis-ci.org/thorgate/tg-utils.svg?branch=master
-        :target: https://travis-ci.org/thorgate/tg-utils
+.. image:: https://github.com/thorgate/tg-react/actions/workflows/python-package.yml/badge.svg?branch=master
```
Comment (jorgenader) [inline README.rst:8]: wrong badge.

**Review by jorgenader — APPROVED** [review]
LGTM. I'll deal any issues that come up.

Thanks.

File: README.rst (hunk 1 above)
Comment (ngnpope) [reply README.rst:8]: Oops, it's a while ago so… 😇

**Comment by ngnpope** [comment]
Thanks Jörgen!

**Pull request merged**